server/Arcadia.Assistant.Feeds/PollsActor.cs

```diff
@@ -0,0 +1,58 @@
+﻿namespace Arcadia.Assistant.Feeds
+{
+    using System;
+    using System.Collections.Generic;
+
+    using Akka.Actor;
+    using Akka.Event;
+    using Akka.Persistence;
+
+    using Arcadia.Assistant.Feeds.Polls;
+    using Arcadia.Assistant.Organization.Abstractions.OrganizationRequests;
+
+    public class PollsActor : UntypedPersistentActor, ILogReceive
```

Comment:
i'm not sure if it's needed. What you need is to add an extra check in feeds actor that returns only messages posted in the certain period